### PR TITLE
Readme updates

### PR DESCRIPTION
**Pull request by tr0njavolta**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,39 @@
-# GitOps ArgoCD Configuration
+# ArgoCD Configuration
 
-GitOps ArgoCD Configuration is reusable Configuration designed to be primarily used in
-higher level Configurations.
+
+This repository contains a [Crossplane configuration](https://docs.crossplane.io/v1.11/concepts/packages/#configuration-packages), tailored for users establishing their initial control plane with [Upbound](https://cloud.upbound.io). This configuration deploys fully managed ArgoCD resources.
```
Comment (haarchri) [inline README.md:4]: https://docs.crossplane.io/latest/concepts/packages/#configuration-packages

File: README.md [hunk 2]
```diff
@@ -1,4 +1,39 @@
...
+
+## Overview
+
+The core components of a custom API in [Crossplane](https://docs.crossplane.io/v1.11/getting-started/introduction/) include:
```
Comment (haarchri) [inline README.md:8]: https://docs.crossplane.io/latest/getting-started/introduction/

File: README.md (hunk 1 above)
Comment (tr0njavolta) [inline README.md:4]:
```suggestion
This repository contains a [Crossplane configuration](https://docs.crossplane.io/latest/concepts/packages/#configuration-packages), tailored for users establishing their initial control plane with [Upbound](https://cloud.upbound.io). This configuration deploys fully managed ArgoCD resources.
```

File: README.md (hunk 2 above)
Comment (tr0njavolta) [inline README.md:8]:
```suggestion
The core components of a custom API in [Crossplane](https://docs.crossplane.io/latest/getting-started/introduction/) include:
```

**Comment by haarchri** [comment]
thanks will change some parts in a followup PR

**Review by haarchri — APPROVED** [review]

**Pull request merged**